modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/ImportTranslationMVCActionCommand.java

```diff
@@ -73,11 +67,7 @@ protected void doProcessAction(
 		ThemeDisplay themeDisplay = (ThemeDisplay)actionRequest.getAttribute(
 			WebKeys.THEME_DISPLAY);
 
-		long articleResourcePrimKey = ParamUtil.getLong(
-			actionRequest, "articleResourcePrimKey");
-		long groupId = ParamUtil.getLong(actionRequest, "groupId");
-		String articleId = ParamUtil.getString(actionRequest, "articleId");
-		double version = ParamUtil.getDouble(actionRequest, "version");
+		JournalArticle article = ActionUtil.getArticle(actionRequest);
```

Comment:
Why this?

Reply:
I did the same in the other pull (see https://github.com/brianchandotcom/liferay-portal/pull/91025/commits/2d1227f33000428cefbcd2f4c4662212c157d572). This is because ActionUtil has more logic that gives you the right article. When the article is a draft we were retrieving the latest publish version instead. So I think it is safer to use this everywhere.